src/components/projects/projectId/labeling/sessionId/main-component/LabelingMainComponent.tsx

```diff
@@ -160,13 +155,15 @@ export default function LabelingMainComponent() {
             return;
         }
         setTimeout(() => {
+            getTokenizedRecord({ recordId: SessionManager.currentRecordId }, (res) => {
+                dispatch(updateRecordRequests('token', res.data.tokenizeRecord));
```

Comment:
getTokenizedRecord  and getRecordByRecordId sent the recordId in a different way

Reply:
won't be changed, personal preference